test/assertions_test.rb

```diff
@@ -149,6 +149,38 @@ def test_assert_statsd_call
     end
   end
 
+  def test_assert_multiple_statsd_calls
+    assert_assertion_triggered do
+      foo_1_metric = StatsD::Instrument::MetricExpectation.new(type: :c, name: 'counter', times: 2, tags: ['foo:1'])
+      foo_2_metric = StatsD::Instrument::MetricExpectation.new(type: :c, name: 'counter', times: 1, tags: ['foo:2'])
+      @test_case.assert_statsd_calls([foo_1_metric, foo_2_metric]) do
+        StatsD.increment('counter', tags: { foo: 1 })
+         StatsD.increment('counter', tags: { foo: 2 })
+      end
+    end
+
+    assert_assertion_triggered do
+      foo_1_metric = StatsD::Instrument::MetricExpectation.new(type: :c, name: 'counter', times: 2, tags: ['foo:1'])
+      foo_2_metric = StatsD::Instrument::MetricExpectation.new(type: :c, name: 'counter', times: 1, tags: ['foo:2'])
+      @test_case.assert_statsd_calls([foo_1_metric, foo_2_metric]) do
+        StatsD.increment('counter', tags: { foo: 1 })
+        StatsD.increment('counter', tags: { foo: 1 })
+        StatsD.increment('counter', tags: { foo: 2 })
+        StatsD.increment('counter', tags: { foo: 2 })
+      end
+    end
+
+    assert_assertion_triggered do
```

Comment:
why does this pass? you are expecting twice foo:1 and once foo:2 and that's what the block does. So shouldn't this be `assert_no_assertion_triggered` then?